src/internal/modules/invoice-accounts/forms/add-fao.js

```diff
@@ -29,7 +31,7 @@ const addFaoForm = (request, selected) => {
         message: 'Select yes if you need to add a person or department as an FAO'
       }
     },
-    choices: [yes, no],
+    choices: [{ value: 'yes', label: 'Yes' }, { value: 'no', label: 'No' }],
```

Comment:
You can use the `options` you defined above:
`choices: [Object.values(options)]`

Reply:
done